src/server/services/user.service.ts

```diff
@@ -0,0 +1,63 @@
+import userRepo from "../repositories/user.repository";
+
+const getOneById = async (id: number) => {
+  const user = await userRepo.getOneById(id);
+  return user;
+};
+const getAll = async () => {
+  try {
+    const result = await userRepo.getAll();
+    return result;
+  } catch (err) {
+    return `Can't load products error:\n ${err.message}`;
```

Comment:
This method has an inconsistent return type. What type of error are you catching here? Most likely you can remove try-catch from here.
The message should also mention "users", not "products".